### ISSUE TITLE
Not enough RAM?

### ISSUE THREAD

**Issue by Orpheliaa** [photopea/photopea#4253]
Hi, everything was going well until randomly (I dont know what triggered it) my photopea started going 'Not enough RAM! (Need 364 MB). I can't save, whenever i click on the other projects/tabs i have it just shows nothing (attachment). Literally anything I press saids I need more RAM including when I try delete projects. it only shows the work i had for the project i was on when it started happening. I tried closing apps and any other tabs but its the same. 
![image](https://user-images.githubusercontent.com/98067742/150247670-2fd90a9c-422d-4a4f-a8c8-095c4575ba07.png)

OK I fixed it that was extremely stressful
basically where it saids bilinear i changed it to like nearest neighbour or something idk what that is but it worked. also the problem started when i was trying to paste 1 project into another project by dragging and dropping if i remember correctly

**Comment by photopea** [comment]
Hi, maybe your resolution is too large, or you have too many layers in opened documents. Save your work regularly, if you know you don't have enough RAM.
